library/rendering/entity_models/src/main/java/org/quiltmc/qsl/rendering/entity_models/api/HasAnimationManager.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.rendering.entity_models.api;
+
+import org.jetbrains.annotations.NotNull;
+import org.quiltmc.qsl.base.api.util.InjectedInterface;
+
+import net.minecraft.client.render.entity.EntityRenderDispatcher;
+import net.minecraft.client.render.entity.EntityRendererFactory;
+
+
+/**
+ * An injected interface for classes that have an {@link AnimationManager}
+ */
+@InjectedInterface(value = {
+		EntityRenderDispatcher.class,
+		EntityRendererFactory.Context.class
+})
+public interface HasAnimationManager {
```

Comment:
If this is meant to be for classes that have an animation manager, shouldn't this class be named `AnimationManagerContainer`?